chrome/content/zotero/bibliography.js

```diff
@@ -149,22 +155,24 @@ var Zotero_File_Interface_Bibliography = new function() {
 		
 		// set style to false, in case this is cancelled
 		_io.style = false;
-	}
-	
+	};
+
+	/*
+	 * Called when locale is changed
+	 */
+	this.localeChanged = function (selectedValue) {
+		selectedLocale = selectedValue;
+	};
+
 	/*
 	 * Called when style is changed
 	 */
-	function styleChanged(index) {
-		// When called from init(), selectedItem isn't yet set
```

Comment:
I think https://github.com/zotero/zotero/pull/623/files#diff-3af4d2a3f7cf404fa9befc1d27591570R99 takes care of this.